core/src/autogluon/core/_setup_utils.py

```diff
@@ -25,6 +27,12 @@
     'tqdm': '>=4.38.0',
     'Pillow': '>=9.0.1,<9.1.0',
     'timm': '>=0.5.4,<0.7.0',
+} if not LITE_MODE else {
+    'numpy': '>=1.21,<1.23',
```

Comment:
why this is more restrictive? We'll have to update this to 1.22.2+
```
-> Vulnerability found in numpy version 1.21.0
   Vulnerability ID: 44715
   Affected spec: <1.22.2
   ADVISORY: Numpy 1.22.2  includes a fix for CVE-2021-41495: Null Pointer Dereference vulnerability exists in numpy.sort in NumPy in the PyArray_DescrNew function due to missing return-value validation,
   which allows attackers to conduct DoS attacks by repetitively creating sort arrays. NOTE: While correct that validation is missing, an error can only occur due to an exhaustion of memory. If the user can...
   CVE-2021-41495
   For more information, please visit https://pyup.io/v/44715/f17
   ```

Reply:
thanks. I merged from the upstream and didn't notice it was upgraded.